diff -ru mercurial-5.3.2~/mercurial/i18n.py mercurial-5.3.2/mercurial/i18n.py
--- mercurial-5.3.2~/mercurial/i18n.py	2020-04-02 09:05:39.000000000 -0700
+++ mercurial-5.3.2/mercurial/i18n.py	2020-04-13 15:12:08.340658201 -0700
@@ -48,7 +48,7 @@
 
 
 datapath = pycompat.fsdecode(resourceutil.datapath)
-localedir = os.path.join(datapath, 'locale')
+localedir = r'/usr/share/locale'
 t = gettextmod.translation('hg', localedir, _languages, fallback=True)
 try:
     _ugettext = t.ugettext
