diff --git a/mercurial/rcutil.py b/mercurial/rcutil.py
--- a/mercurial/rcutil.py
+++ b/mercurial/rcutil.py
@@ -22,6 +22,7 @@
 
 fallbackpager = scmplatform.fallbackpager
 systemrcpath = scmplatform.systemrcpath
+clearrcpath = scmplatform.clearrcpath
 userrcpath = scmplatform.userrcpath
 
 
@@ -94,7 +95,7 @@
         normpaths = lambda paths: [
             (b'path', os.path.normpath(p)) for p in paths
         ]
-        _rccomponents = normpaths(defaultrcpath() + systemrcpath())
+        _rccomponents = normpaths(defaultrcpath() + systemrcpath() + clearrcpath())
         _rccomponents.append(envrc)
         _rccomponents.extend(normpaths(userrcpath()))
     return _rccomponents
diff --git a/mercurial/scmposix.py b/mercurial/scmposix.py
--- a/mercurial/scmposix.py
+++ b/mercurial/scmposix.py
@@ -51,6 +51,18 @@
     return path
 
 
+def clearrcpath():
+    path = []
+    root = b'/usr/share/defaults/mercurial'
+    # old mod_python does not set sys.argv
+    if len(getattr(sys, 'argv', [])) > 0:
+        p = os.path.dirname(os.path.dirname(pycompat.sysargv[0]))
+        if p != b'/':
+            path.extend(_rcfiles(os.path.join(p, root)))
+    path.extend(_rcfiles(b'/' + root))
+    return path
+
+
 def userrcpath():
     if pycompat.sysplatform == b'plan9':
         return [encoding.environ[b'home'] + b'/lib/hgrc']
