diff -ru mercurial-5.3.2~/mercurial/rcutil.py mercurial-5.3.2/mercurial/rcutil.py
--- mercurial-5.3.2~/mercurial/rcutil.py	2020-04-02 09:05:39.000000000 -0700
+++ mercurial-5.3.2/mercurial/rcutil.py	2020-04-13 17:19:07.324010738 -0700
@@ -24,6 +24,7 @@
 
 fallbackpager = scmplatform.fallbackpager
 systemrcpath = scmplatform.systemrcpath
+clearrcpath = scmplatform.clearrcpath
 userrcpath = scmplatform.userrcpath
 
 
@@ -102,6 +103,7 @@
             (b'path', os.path.normpath(p)) for p in paths
         ]
         _rccomponents.extend(normpaths(systemrcpath()))
+        _rccomponents.extend(normpaths(clearrcpath()))
         _rccomponents.append(envrc)
         _rccomponents.extend(normpaths(userrcpath()))
     return _rccomponents
diff -ru mercurial-5.3.2~/mercurial/scmposix.py mercurial-5.3.2/mercurial/scmposix.py
--- mercurial-5.3.2~/mercurial/scmposix.py	2020-04-02 09:05:39.000000000 -0700
+++ mercurial-5.3.2/mercurial/scmposix.py	2020-04-13 17:20:57.003109516 -0700
@@ -51,6 +51,18 @@
     return path
 
 
+def clearrcpath():
+    path = []
+    root = b'/usr/share/defaults/mercurial'
+    # old mod_python does not set sys.argv
+    if len(getattr(sys, 'argv', [])) > 0:
+        p = os.path.dirname(os.path.dirname(pycompat.sysargv[0]))
+        if p != b'/':
+            path.extend(_rcfiles(os.path.join(p, root)))
+    path.extend(_rcfiles(b'/' + root))
+    return path
+
+
 def userrcpath():
     if pycompat.sysplatform == b'plan9':
         return [encoding.environ[b'home'] + b'/lib/hgrc']
