diff --git a/mercurial/rcutil.py b/mercurial/rcutil.py
index 38fe8eb..a2b572d 100644
--- a/mercurial/rcutil.py
+++ b/mercurial/rcutil.py
@@ -24,6 +24,7 @@ else:
 
 fallbackpager = scmplatform.fallbackpager
 systemrcpath = scmplatform.systemrcpath
+clearrcpath = scmplatform.clearrcpath
 userrcpath = scmplatform.userrcpath
 
 
@@ -102,6 +103,7 @@ def rccomponents():
             (b'path', os.path.normpath(p)) for p in paths
         ]
         _rccomponents.extend(normpaths(systemrcpath()))
+        _rccomponents.extend(normpaths(clearrcpath()))
         _rccomponents.append(envrc)
         _rccomponents.extend(normpaths(userrcpath()))
     return _rccomponents
diff --git a/mercurial/scmposix.py b/mercurial/scmposix.py
index 76fe8be..d9b9e07 100644
--- a/mercurial/scmposix.py
+++ b/mercurial/scmposix.py
@@ -51,6 +51,18 @@ def systemrcpath():
     return path
 
 
+def clearrcpath():
+    path = []
+    root = b'/usr/share/defaults/mercurial'
+    # old mod_python does not set sys.argv
+    if len(getattr(sys, 'argv', [])) > 0:
+        p = os.path.dirname(os.path.dirname(pycompat.sysargv[0]))
+        if p != b'/':
+            path.extend(_rcfiles(os.path.join(p, root)))
+    path.extend(_rcfiles(b'/' + root))
+    return path
+
+
 def userrcpath():
     if pycompat.sysplatform == b'plan9':
         return [encoding.environ[b'home'] + b'/lib/hgrc']
